src/main/java/com/gradle/CommonCustomUserDataGradlePlugin.java

```diff
@@ -146,7 +146,7 @@ private void applyProjectPlugin(Project project, DevelocityAdapter develocity) {
 
         BuildScanAdapter buildScan = develocity.getBuildScan();
         customDevelocityConfig.configureBuildScanPublishing(buildScan);
-        CustomBuildScanEnhancements buildScanEnhancements = new CustomBuildScanEnhancements(develocity, providers, project.getGradle());
+        CustomBuildScanEnhancements buildScanEnhancements = new CustomBuildScanEnhancements(develocity, providers, project.getGradle(), project.getRootDir().getAbsolutePath());
```

Comment:
Nitpick, but stylistically, I'd lean towards passing just the rootDir here as a a file, all the way into `execAndGetStdOut` have that be the only place where we need to care about `-C`